lib/site_settings/validations.rb

```diff
@@ -201,6 +201,13 @@ def validate_cors_origins(new_val)
     validate_error :cors_origins_should_not_have_trailing_slash
   end
 
+  def validate_slow_down_crawler_user_agents(new_val)
+    return if new_val.blank?
+    if new_val.split("|").any? { |crawler| crawler.size < 3 }
+      validate_error :slow_down_crawler_user_agent_must_be_at_least_3_characters
```

Comment:
I see about wanting the extra validation here, I guess we can test on  not matching `["gecko","mozilla","firefox","khtml","windows","linux","apple","webkit"]` and a few other obvious strings.

Reply:
done!